src/components/TimePicker/TimePicker.tsx

```diff
@@ -452,58 +455,52 @@ function TimePicker({forwardedRef, defaultValue, onSubmit, onInputChange}) {
                 >
                     <AmountTextInput
                         placeholder={numberFormat(0)}
-                        maxLength={2}
```

Comment:
This prop doesn't exist? Maybe it's missing in AmountTextInput types?

Reply:
Yep